### PR TITLE
Create Gspidey123.md

### PR DESCRIPTION
**Pull request by Gspidey123**
## Describe your project in 1-3 sentences

 I am building a mechatronic hand that is 3D printed that uses nitinol springs for muscles and is controlled with an Arduino. 

## Hack Club Slack handle

Username: Gspidey123
U04KPQMDR6C 

## Checklist before submitting proposal:

- [y] I am a current high school student (or younger).
- [y] I have submitted an RSVP at <https://hackclub.com/winter>
- [y] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [y] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! **we're back on tuesday and will start reviewing PRs in the order that they came in, thanks for your patience :)**

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts & ready software

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: this sounds super cool! i'm wondering, are you referencing any sort of guides for this and does your budget include shipping and taxes?)_

our Jan 15 deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

**Comment by Gspidey123** [comment]
Hi Belle,

I have double checked and think the file is properly formatted. I also have edited the PR to add some images and more detail to make it easier to understand how my project works. I have included shipping in the costs. Most of these are from Amazon Prime, so has free delivery. And I have included the delivery for the Nitinol springs. All the money is in pounds converted to US dollars.

The main hand will be 3D printed using an online file. However, I want try and innovate the field by using a shape memory alloy, so for the rest of it, I am not following a guide and am very much experimenting in the unknown. However, my older brother has more experience with electronics and materials, and he will be helping me if it gets too technical for me. 

Thanks,
Gokul

**Comment by bellesea** [comment]
@Gspidey123 thanks so much for the additional context! and, a heads up that we've started reviewing projects but due to the flurry last weekend (100+ PRs ksdjhfkjdshf) it might take a while before we review yours, thank you for your patience!

File: Gspidey123.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: "@Gspidey123"
+project: "Mechatronic Hand"
+---
+ 
+# Project Name
```
Comment (exu3) [inline Gspidey123.md:6]: Could you update this to say your project name instead of "project name"?

File: Gspidey123.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Plan
+Firstly, I can use a simple string-based hand that can be 3D printed. My brother bought a 3d printer for the house, so I can use an existing file to do this. I have attached a picture of an example hand file from thingiverse, which is what I will probably use. However, I will use fishing wire as a stronger alternative to string for the tendons. 
+
+![image](https://user-images.githubusercontent.com/122729898/212999254-8a3df9b5-cd98-42a0-994e-6cc87d7454af.png)
+
+
+Since this requires very precise 3D printing, I would like to first upgrade the 3D printer to have automatic bed levelling to make this possible (as my 3D printer currently faces some acuraccy issues such as layer distortion and will not be capable of it ). I have already bought some cheap nitinol wire and made a quick model using simply a switch and a battery (picture shown below).
+
+![WhatsApp Image 2023-01-15 at 15 10 54](https://user-images.githubusercontent.com/122729898/213000244-588885e6-2c4d-42c0-8eb8-2a77f40758a7.jpeg)
```
Comment (exu3) [inline Gspidey123.md:21]: Could you update the Alt text here to describe what we see in the image?

File: Gspidey123.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+It showed potential, but the contraction was not enough for the full range of motion. So, I instead plan to use nitinol springs to act as the muscles of the hand (which allows for a much greater contraction) and an Arduino microcontroller to make it fully automatic (by using it as multiple electronic switches).  I can also add things like a joystick to control it, or a camera with a program that allows it to mirror my hand. Some extra functions for the hand I plan are to make it solar powered if possible (by using some solar cells as the power source), to promote sustainable energy usage. And I want it to have a child friendly aesthetic by using rainbow colours. Also, I can use an Arduino Wi-fi module to connect this to a controller or camera or simply to the computer (all of which I already own). 
+ 
+ 
+## Budget
+ 
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
```
Comment (exu3) [inline Gspidey123.md:29]:
```suggestion
```

File: Gspidey123.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Mechatronic Hand
+ 
+## Summary
+ 
+I recently learnt of the existence of smart materials such as nitinol, and when I did a quick search, was shocked to find that it was accessible to me. This made me wonder why I haven’t seen it in use more. Many people have attempted making moveable electronic hands, however, I wanted to try something new in this field by using nitinol to electronically control the movement.  I want to experiment to see if this could be the future of bionics. 
+
+## Plan
+Firstly, I can use a simple string-based hand that can be 3D printed. My brother bought a 3d printer for the house, so I can use an existing file to do this. I have attached a picture of an example hand file from thingiverse, which is what I will probably use. However, I will use fishing wire as a stronger alternative to string for the tendons. 
+
+![image](https://user-images.githubusercontent.com/122729898/212999254-8a3df9b5-cd98-42a0-994e-6cc87d7454af.png)
```
Comment (exu3) [inline Gspidey123.md:16]: Could you update the alt text here to describe what a user sees in the image?

**Review by exu3 — CHANGES_REQUESTED** [review]
I love the photos you included. I left some comments below. It's looking good so far!

**Review by exu3 — APPROVED** [review]
Once the comments above are addressed, your project will be good to go! 

The @hackclub/bank-ops team will DM you on Slack with next steps for receiving the grant.

**Comment by Gspidey123** [comment]
Thank you so much! I have edited the file and addressed all the comments.

**Review by MelSmith104 — COMMENTED** [review]
hi @Gspidey123! this project sounds fascinating! i remember hearing about nitinol and also thought it was super cool! im dm'ing you right now to chat about next steps!

**Comment by Gspidey123** [comment]
I'm really sorry but I cant seem to find the dm that you sent me. Will it be on slack?

**Comment by MelSmith104** [comment]
just dm'ed you again! im so sorry! idk why my last one didn't send :(

**Review by MelSmith104 — APPROVED** [review]

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**